Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
```

Comment:
### [net-scp](https://github.com/net-ssh/net-scp)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 3.0.0 → 4.0.0.rc1

[_[change-log](https://github.com/net-ssh/net-scp/blob/master/CHANGES.txt), ~~source-code~~, [gem-diff](https://my.diffend.io/gems/net-scp/3.0.0/4.0.0.rc1)_]





<details>
<summary>Commits</summary>

A change of **4** commits. See the full changes on [the compare page](https://github.com/net-ssh/net-scp/compare/v3.0.0...v4.0.0.rc1).

These are the individual commits:
- (9b5bac5) [net-ssh 7.* support](https://github.com/net-ssh/net-scp/commit/9b5bac50dcd59c349e82e5322e6bc307f4efceb9)
- (e6bec09) [Merge pull request #60 from hana-da/net-ssh-7](https://github.com/net-ssh/net-scp/commit/e6bec090768c5d498042b44089856e1abe1a5ad4)
- (9e8db35) [Update version.rb](https://github.com/net-ssh/net-scp/commit/9e8db351c7a16bd14108115a839be8f5822dc6b8)
- (85b340b) [Updated cert](https://github.com/net-ssh/net-scp/commit/85b340be0347954776e20e3271764b25791dc5d9)


</details>